tests/custom.rs

```diff
@@ -1,54 +1,43 @@
-// Test that a custom handler works on wasm32-unknown-unknown
-#![cfg(all(
-    target_arch = "wasm32",
-    target_os = "unknown",
-    feature = "custom",
-    not(feature = "js")
-))]
-
-use wasm_bindgen_test::wasm_bindgen_test as test;
```

Comment:
I think we should keep this, but do it this way, which is commonly done in other crates:
```
#[cfg(all(target_arch = "wasm32", target_os = "unknown"))]
use wasm_bindgen_test::wasm_bindgen_test as test;
```
This way, all tests marked `#[test]` will be run using either libtest or wasm_bindgen_test.

Reply:
Done.